### PR TITLE
[opentelemetry-cpp] update version to v1.1.1

### PR DESCRIPTION
**Pull request by juandemanjon**
Bumping opentelemetry-cpp version

### CODE REVIEW

File: ports/opentelemetry-cpp/vcpkg.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
   "name": "opentelemetry-cpp",
-  "version-semver": "1.0.1",
-  "port-version": 1,
+  "version-semver": "1.1.1",
+  "port-version": 2,
```
Comment (JonLiu1993) [inline ports/opentelemetry-cpp/vcpkg.json:5]:
```suggestion
```

**Comment by JonLiu1993** [comment]
@juandemanjon ,Have you tesed features locally?

**Comment by juandemanjon** [comment]
@JonLiu1993 I did tested locally jaeger, zipkin, and otlp features only on linux with ```./vcpkg install opentelemetry-cpp[jaeger,zipkin,otlp]``` command

**Comment by JonLiu1993** [comment]
> @JonLiu1993 I did tested locally jaeger, zipkin, and otlp features only on linux with `./vcpkg install opentelemetry-cpp[jaeger,zipkin,otlp]` command

Please apply my review suggestion

File: ports/opentelemetry-cpp/vcpkg.json (hunk 1 above)
Comment (juandemanjon) [reply ports/opentelemetry-cpp/vcpkg.json:5]: @JonLiu1993 Why are you suggesting to remove "port-version"? I think, it is required

Comment (JonLiu1993) [reply ports/opentelemetry-cpp/vcpkg.json:5]: @juandemanjon ，When the version of the underlying library is upgraded, this field should be set back to 0, you can refer to the [maintainers](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/control-files.md#port-version)

Comment (juandemanjon) [reply ports/opentelemetry-cpp/vcpkg.json]: @JonLiu1993 I have committed your suggested change

**Review by JonLiu1993 — APPROVED** [review]

File: ports/opentelemetry-cpp/vcpkg.json (hunk 1 above)
Comment (juandemanjon) [reply ports/opentelemetry-cpp/vcpkg.json]: @JonLiu1993 Follow up question. "port-version" has been removed. Does it default to zero? Or we should explicitly set to zero

Comment (JonLiu1993) [reply ports/opentelemetry-cpp/vcpkg.json]: it default to zero

**Comment by ras0219-msft** [comment]
Thanks for the update!

**Pull request merged**